supervision/detection/utils.py

```diff
@@ -48,6 +48,53 @@ def polygon_to_mask(polygon: np.ndarray, resolution_wh: Tuple[int, int]) -> np.n
     return mask
 
 
+def box_iou(
+    box_true: Union[List[float], np.ndarray],
+    box_detection: Union[List[float], np.ndarray],
+) -> float:
+    """
+    Compute the Intersection over Union (IoU) between two bounding boxes.
+
+    Both `box_true` and `box_detection` should be in (x_min, y_min, x_max, y_max)
+    format.
+
+    Note:
+        Use `box_iou` when computing IoU between two individual boxes.
+        For comparing multiple boxes (arrays of boxes), use `box_iou_batch` for better
+        performance.
```

Comment:
IMO, a similar note should be present in the docstring of `box_iou_batch` as well.